src/profiles.js

```diff
@@ -0,0 +1,199 @@
+import tv4 from 'tv4'
+import jsonlint from 'json-lint'
+import _ from 'lodash'
+import path from 'path'
+
+import Utils from './utils'
+
+const DEFAULT_REMOTE_PATH = 'http://schemas.datapackages.org/registry.csv'
+let DEFAULT_LOCAL_PATH
+if (!Utils.isBrowser) {
+  DEFAULT_LOCAL_PATH = path.join(__dirname, '..', 'schemas', 'registry.csv')
+}
+
+class Profiles {
+
+  constructor(remote = false) {
+    const self = this
+        , PATH = remote || Utils.isBrowser ? DEFAULT_REMOTE_PATH :
```

Comment:
I suppose it will be much more readable:
`( remote || Utils.isBrowser ) ? DEFAULT_REMOTE_PATH : DEFAULT_LOCAL_PATH`